### DISCUSSION TITLE
Question regarding columns of make_future_dataframe()

### DISCUSSION THREAD

**Discussion by aiwalter** [ourownstory/neural_prophet#267] [Q&A - get help using NeuralProphet]
How is the value `t` calculated when doing `m.make_future_dataframe(df, periods=30)`? And what is it used for? Also why is it required to provide also `y`? In `fbprophet`, it is possible do to predictions only on the `ds` column, would be great of this would also be possible for `neuralprophet` :)

![image](https://user-images.githubusercontent.com/29627036/104007763-ba7c5c80-51a8-11eb-9a3d-0c93d7bf4545.png)

Thank you!

**Comment by HansikaPH** [comment]
@aiwalter The t value is computed and added to the data frame both during model fitting and prediction. The t value is basically a min-max normalized version of the ds column where the min is the start date of the data frame and the scale is the total number of days considered. This t value is used internally to model the trend. 

Once the model is fitted on the training data, we provide the support to make predictions using a completely new data frame. In that case, the model is fitted on a separate set of training data, but during prediction, the model will be fed with new testing data (with different dates) to predict its future. This is different from the original Prophet, where the predictions can be made only for the future of the training data. This is why we have given the support to provide new `y` values with new `ds` into the `make_future_dataframe` function.

**Reply by sivikt** [reply]
@HansikaPH @judussoari  Hi! What would you advise in the case if we need predictions for more than n_forecasts value? Sometimes there is a case. Could we `make_future_dataframe` for the first n_forecasts data points, predict and then feed the predictions into the history and create another future for the next n_forecasts data points? Does it make sense?
